src/views/Main/Login/Login.js

```diff
@@ -20,9 +20,10 @@ export class Login extends React.Component {
 
   render() {
     return (
-      <div className={styles.root}>
+      <div className={'root'}>
+        <img src="https://cdn.auth0.com/styleguide/1.0.0/img/badge.svg" role="presentation" />
```

Comment:
I made a constant, but these will all go away once we have the logo. Just acting as a placeholder for now